runtime/common/src/xcm_sender.rs

```diff
@@ -19,7 +19,7 @@
 use parity_scale_codec::Encode;
 use runtime_parachains::{configuration, dmp};
 use sp_std::marker::PhantomData;
-use xcm::opaque::v1::{Error, Junction, MultiLocation, Result, SendXcm, Xcm};
+use xcm::opaque::v1::{Error, Junction, Junctions, MultiLocation, Result, SendXcm, Xcm};
```

Comment:
probably can be simplified to

```suggestion
use xcm::latest::prelude::*;
```